### ISSUE TITLE
Group referrers by domain

### ISSUE THREAD

**Issue by lucasdcrk** [electerious/Ackee#114]
Hi!

I think it would be cool to group the referrers by domains and possibly exclude those that are the same domain.
![4ZJd.png](https://i.apo.pm/s/4ZJd.png)
As you can see this section is flooded with same-site referrers

**Comment by electerious** [comment]
They might have the same domain, but I guess all with different pages? Tracking the pages (and not only the domains) is usually helpful. Image only tracking twitter.com. You wouldn't be able to see who has tweeted about your page/

**Comment by lucasdcrk** [comment]
Yeah I know this is important to keep the pages but I think it would be cleaner to display a list of domains and when you click on it it gives detailed infos (namely the pages). Also, any idea how to exclude the same domain? In my case in need to scroll quite a bit to see useful data (Twitter links, ...).

**Comment by MatthK** [comment]
I second that.

I would go a step further. One area should show external referers, so you see who brings you traffic. 
The "internal" referers could be used to show a flow diagram on how the users navigate through the sites (Homepage -> Page 1 -> Page 2 -> Exit).

**Comment by electerious** [comment]
@lucasdcrk You could [overwrite the siteReferrer](https://github.com/electerious/Ackee/issues/113#issuecomment-633971352) until there's a better way.
